pkg/cvo/cvo_scenarios_test.go

```diff
@@ -1566,6 +1566,253 @@ func TestCVO_UpgradeUnverifiedPayload(t *testing.T) {
 	})
 }
 
+func TestCVO_ResetPayloadLoadStatus(t *testing.T) {
+	o, cvs, client, _, shutdownFn := setupCVOTest("testdata/payloadtest-2")
+
+	// Setup: a successful sync from a previous run, and the operator at the same image as before
+	//
```

Comment:
> This can be removed.

The extra comment line? If so I'm just following the convention already established in the file. Removing these would be extensive.